README.md

```diff
@@ -185,11 +185,9 @@ To publish locally, you'll need to do the following...
 ####Sign In
 
 To publish locally, you'll need an account.
-* Go to [http://localhost:3000/account](http://localhost:3000/account)
-* Click ``Join Webmaker`` and complete the process, you can use a fake email
-* When you've created your account, click ``Set permanent password instead``
-  * This lets you authenticate your account without needing email
-* Go back to Thimble and Log In with your new account
+* At the top right corner of the Thimble main screen  Click  ``Sign In`` if you have an account or click ``Create an account`` and complete the process, you can use a fake email
```

Comment:
Still need change to `{space}{space}Click{space}{space}` to just be `{space}click{space}` (notice lower case)